traffic_portal/app/src/common/modules/widget/cdnChart/WidgetCDNChartController.js

```diff
@@ -138,9 +138,9 @@ var WidgetCDNChartController = function(cdn, $scope, $timeout, $filter, $q, $int
 			},
 			tooltip: {
 				show: true,
-				content: function(label, xval, yval, flotItem){
+				content: function(label, xval, yval, floatItem){
```

Comment:
so this isn't a typo. https://www.flotcharts.org/

Reply:
I knew that was a gamble, should have asked first haha.